src/v2/view-builder/views/ov/ChallengeOktaVerifyFastPassView.js

```diff
@@ -47,7 +47,12 @@ const Body = BaseForm.extend(Object.assign(
       return this.options.currentViewState.relatesTo.value.contextualData.challenge.value;
     },
 
-    doLoopback(authenticatorDomainUrl = '', ports = [], challengeRequest = '') {
+    doLoopback(deviceChallenge) {
+      let authenticatorDomainUrl = deviceChallenge.domain !== undefined ? deviceChallenge.domain : '';
```

Comment:
A neat JS feature this repo supports is the [nullish coalescing operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator). You can simplify checking for undefined via:

```suggestion
      const authenticatorDomainUrl = deviceChallenge.domain ?? '';
```